drake_cmake_installed/README.md

```diff
@@ -2,24 +2,48 @@
 
 This uses the CMake `find_package(drake)` mechanism to find an installed instance of Drake.
 
-# Locally on Xenial
+# Instructions
 
-Download the binaries and install into /opt. This will ensure that CMake is able
-to auto-discover the `drake-config.cmake` module. 
+These instructions are only supported for Ubuntu 16.04 (Xenial).
 
 ```
-# A nightly build
+# Install PreRequisites
+<TODO: install pre-requisite debs list>
+
+# Install Drake to /opt/drake
 # curl -O https://s3.amazonaws.com/drake-packages/drake/nightly/drake-20170719-xenial.tar.gz
 # Alternatively, the latest (usually last night's build)
 curl -O https://s3.amazonaws.com/drake-packages/drake/nightly/drake-latest-xenial.tar.gz
 sudo tar -xvzf drake-latest-xenial.tar.gz -C /opt
-mkdir build
-cd build
+
+# Install GTest
+sudo apt-get install libgtest-dev
+cd /usr/src/gtest
+sudo mkdir build && cd build && sudo cmake .. && sudo make
+sudo cp *.a /usr/lib
```

Comment:
Not sure this is the approved Ubuntu way of doing things.

Reply:
I am going to stand fast against using `sudo` to modify the Debian-owned pieces of the filesystem outside of `apt`'s purview.  I haven't found the relevant section in the "here" link, but if there is a way to write CMakeLists to do it, that seems strictly superior, even if it means more boilerplate for mimic projects to duplicate.

Reply:
Certainly superior, but I would claim it is not Drake's responsibility to teach the world how to install and use gtest. Nor is it a subproject I could justify company time to commit to.

I have updated the particularly offensive (to me also) sudo build instructions in /usr/src/gtest and inserted a link that shambhala users can follow if they want to learn how to explicitly bring gtest into the build themselves. 

If there was someone who could rapidly produce it from CMake code they already have, great. If someone contributed a PR for us on their own time, great. I might kick back on these because they complicate the example though. If that was the case, simple ctests here might suffice best of all.

Summary: If you really feel strongly to follow up on any of the above, let's follow it up in another issue/PR.